examples/002_EngagingConsistently/002_EngagingConsistently.cpp

```diff
@@ -287,6 +272,24 @@ bool playEngagingConsistently() {
     }
   }
 
+  // Send report
+  Log.info("Sending report");
+  String extra = String::format(
+      "{\"pos_tries\":%u,\"neg_tries\":%u", countSuccesses(), countMisses());
+  if (challengeComplete) {extra += ",\"challengeComplete\":1";};
```

Comment:
It looks like there's one too many semicolons on this line.

Reply:
Woops, seems to work with the extra one, but fixed it anyway!